algorithms/color_refinement.py

```diff
@@ -36,30 +41,38 @@ def __initialize_colors(G: "Graph"):
     return colors, max_colornum
 
 
-def __colorgroup_refinement(colors, vertices: List["Vertex"], max_colornum):
+def __colorgroup_refinement(colors, colornum, vertices: List["Vertex"], next_colornum):
```

Comment:
Hier staan nog niet alle types bij die we returnen, ik zou het denk ik of allemaal wel doen of allemaal niet. Maar het maakt mij niet uit wat het wordt ;)